internal/proposals/logs/logs-stream-destination-and-library-recomendation.md

```diff
@@ -84,6 +86,7 @@ Pros:
 
 Cons:
 
+- Lack of `Caller` field
```

Comment:
```suggestion
- No `Caller` field
```
There's a subtle difference between `lack of` and `no`. `Lack of` means that there's not enough (of) something, or something isn't available (like, lack of money preventing you from doing sth). 
What you mean here, I believe, is just stating the fact that this library doesn't have this field.